### PR TITLE
First pass over "Practical information".

### PR DESCRIPTION
**Pull request by terjekv**
Apologies for the "Merge branch"-commits.  :-(

### CODE REVIEW

File: docs/practical_information/README.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+organisers of the tutorial, other participants, and volunteers. Please use this channel to ask
+questions throughout the tutorial, or for any other assistance related to the tutorial.
 
-To connect, you will need to create an account in the EasyBuild Slack first,
-which you can do via [https://easybuild-slack.herokuapp.com/](https://easybuild-slack.herokuapp.com/).
+To access this channel, you will first need to create an account in the EasyBuild Slack.
+This is done via [https://easybuild-slack.herokuapp.com/](https://easybuild-slack.herokuapp.com/).
 
-Once you have an account, you can join the EasyBuild Slack via [https://easybuild.slack.com/](https://easybuild.slack.com/), and then join the `#tutorial` channel.
+Once you have the account set up, you can join the EasyBuild Slack via [https://easybuild.slack.com/](https://easybuild.slack.com/). From there you can the `#tutorial` channel.
```
Comment (boegel) [inline docs/practical_information/README.md:15]:
```suggestion
Once you have the account set up, you can join the EasyBuild Slack via [https://easybuild.slack.com/](https://easybuild.slack.com/). From there you can join the `#tutorial` channel.
```

File: docs/practical_information/README.md [hunk 2]
```diff
@@ -51,6 +55,8 @@ and will be downloaded automatically from Docker Hub when you run the `docker` o
 
 #### Using Docker
 
+Please note that docker requires root access, which is available to you in the tutorials AWS environment.
```
Comment (boegel) [inline docs/practical_information/README.md:58]: No, `docker` will be pre-installed in the AWS environment, but you don't need root in the AWS env at all (the Docker daemon runs as root, but that's irrelevant to the tutorial participants).

File: docs/practical_information/README.md (hunk 1 above)
Comment (terjekv) [reply docs/practical_information/README.md]: Thanks!

File: docs/practical_information/README.md (hunk 2 above)
Comment (terjekv) [reply docs/practical_information/README.md]: Context brainfart. Thanks. Removed.

**Pull request merged**